keyboards/argo_works/ishi/80/mk0_avr_extra/rules.mk

```diff
@@ -0,0 +1,4 @@
+OLED_ENABLE = yes
+OLED_DRIVER = SSD1306
+WPM_ENABLE = yes
+PIN_COMPATIBLE = elite_c
```

Comment:
```suggestion
```

Reply:
What is the reasoning behind removing this line?

I have included it as this specific version of the firmware is made for daughterboards with the 5 pins on the bottom (on which some keyboard functionality depends) and so while using the converter it'll be necessary.

Reply:
@waffle87 When the pin compatibility is specified in the info.json file alone and firmware compilation to the Elite-Pi is attempted it throws the following error

<img width="960" alt="image" src="https://user-images.githubusercontent.com/40370113/208743947-ea30a5b0-db71-4861-8089-d4b52e43d2c9.png">

The same behaviour is observed when converting to the Stemcell. When I specify pin compatibility in rules.mk alone or in addition to the information in info.json, there is no error and the firmware compiles successfully.

Reply:
Ah I see — seems like if `pin_compatible` is set to `elite_c` in info.json, it still defaults to `promicro` (this can be observed in `.build/obj_mk0_avr_extra/src/info_rules.mk`) so that's a bug elsewhere. This is fine to leave for now.